sdk/script/acquisition-sdk.ts

```diff
@@ -46,17 +46,29 @@ export interface LocalPackage extends Package {
 export interface Callback<T> { (error: Error, parameter: T): void; }
 
 export interface Configuration {
-    serverUrl: string;
+    appVersion: string;
+    clientUniqueID: string;
     deploymentKey: string;
+    serverUrl: string;
     ignoreAppVersion?: boolean
 }
 
+interface DeploymentStatusReport {
```

Comment:
Is this interface synchronized with the server? Would it be worth adding this to rest-definitions?